lib/styles/fonts.dart

```diff
@@ -51,6 +51,49 @@ class AppTextStyles {
     fontSize: 16,
   );
 
+  // Mobile-specific headings
+  static const TextStyle heading1Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 24, // Smaller font size for mobile
+  );
+
+  static const TextStyle heading2Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 22, // Smaller for mobile
+  );
+
+  static const TextStyle heading3Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 20,
+  );
+
+  static const TextStyle heading4Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 18,
+  );
+
+  static const TextStyle heading5Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 16,
+  );
+
+  static const TextStyle heading6Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 14,
+  );
+
```

Comment:
Restore previous headings for desktop here

Reply:
**I HAVE MADE THE STARED CHANGES.
AS WANTED, THE DESKTOP HEADINGS HAVE BEEN RESTORED**

Reply:
**THE PREVIOUS DESKTOP HEADINGS HAVE BEEN RESTORED**